development-defaults.ini

```diff
@@ -151,10 +151,6 @@ no_shirt     = 0
 
 [enums]
 
```

Comment:
Are you sure this wasn't part of some special-casing? I think these were left in because they prevented 500 errors.